[DO NOT REVIEW] TESTING

pritamdamania87 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#31812 [DO NOT REVIEW] TESTING**
* #31811 [DO NOT REVIEW] TESTING





Differential Revision: [D19269559](https://our.internmc.facebook.com/intern/diff/D19269559/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D19269559/)!

Differential Revision: [D19269559](https://our.internmc.facebook.com/intern/diff/D19269559)

pritamdamania committed:
[DO NOT REVIEW] TESTING



Differential Revision: [D19269559](https://our.internmc.facebook.com/intern/diff/D19269559/)

**NOTE FOR REVIEWERS**: This PR has internal facebook specific changes, please review them on [phabricator](https://our.internmc.facebook.com/intern/diff/D19269559/)!

[ghstack-poisoned]

pritamdamania committed:
Update on "[DO NOT REVIEW] TESTING"


Differential Revision: [D19269559](https://our.internmc.facebook.com/intern/diff/D19269559/)

**NOTE FOR REVIEWERS**: This PR has internal facebook specific changes, please review them on [phabricator](https://our.internmc.facebook.com/intern/diff/D19269559/)!

[ghstack-poisoned]

pritamdamania committed:
Update on "[DO NOT REVIEW] TESTING"






Differential Revision: [D19269559](https://our.internmc.facebook.com/intern/diff/D19269559/)

**NOTE FOR REVIEWERS**: This PR has internal facebook specific changes, please review them on [phabricator](https://our.internmc.facebook.com/intern/diff/D19269559/)!

Differential Revision: [D19269559](https://our.internmc.facebook.com/intern/diff/D19269559)

[ghstack-poisoned]

pritamdamania committed:
Update on "[DO NOT REVIEW] TESTING"






Differential Revision: [D19269559](https://our.internmc.facebook.com/intern/diff/D19269559/)

**NOTE FOR REVIEWERS**: This PR has internal facebook specific changes, please review them on [phabricator](https://our.internmc.facebook.com/intern/diff/D19269559/)!

Differential Revision: [D19269559](https://our.internmc.facebook.com/intern/diff/D19269559)

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit effe7993:


**None of the build failures appear to be your fault**.

* **1/1** [broken upstream](https://dr.pytorch.org/master-timeline.html?count=1&sha1=ed5cd0d7429651fd534edc04567d890cfb5eec0d&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35) at merge base ed5cd0d7 since Jan 02
  <details>
  <summary>
  You may want to rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch origin viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch origin viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=effe799385cba080b0e25c48bbb4a516eca202cc).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably **caused by upstream breakages**:
* [pytorch_macos_10_13_py3_test](https://dr.pytorch.org/build-details.html?build_id=79616435) from Jan 02 until Jan 02



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

pritamdamania committed:
Update on "[DO NOT REVIEW] TESTING"






Differential Revision: [D19269559](https://our.internmc.facebook.com/intern/diff/D19269559/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D19269559/)!

Differential Revision: [D19269559](https://our.internmc.facebook.com/intern/diff/D19269559)

[ghstack-poisoned]
